src/Sylius/Behat/spec/Context/Transform/CountryContextSpec.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Addressing\Converter\CountryNameConverterInterface;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class CountryContextSpec extends ObjectBehavior
+{
+    function let(
+        RepositoryInterface $countryRepository,
+        FactoryInterface $countryFactory,
+        CountryNameConverterInterface $countryNameConverter
+    ) {
+        $this->beConstructedWith($countryRepository, $countryFactory, $countryNameConverter);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Behat\Context\Transform\CountryContext');
+    }
+
+    function it_implements_context_interface()
```

Comment:
We should be consistent and use one convention - [here](https://github.com/Sylius/Sylius/pull/4442/files#diff-cd537122d1dbf99f282cb0fafe1c6ec8R35) you use ``it_is_context`` and here ``it_implements_context_interface``. We're not consistent with this in *Sylius*, lets keep convention in a PR at least :smile: